ted_sws/rdf_differ/services/difference_between_rdf_files.py

```diff
@@ -0,0 +1,38 @@
+from typing import List
+
+import rdflib
+from rdflib import Graph
+from rdflib.compare import isomorphic, graph_diff
+
+from tests import TEST_DATA_PATH
+
+FIRST_RML_FILE = TEST_DATA_PATH / "technical_mapping_F03.rml.ttl"
+SECOND_RML_FILE = TEST_DATA_PATH / "technical_mapping_F06.rml.ttl"
+
+
+def rdf_differ_service(FIRST_RML_FILE: str, SECOND_RML_FILE: str):
```

Comment:
I see some problems here with functions formal arguments:
1. For function formal arguments use only snake case( like this: my_var);
2. Don't use same name with global vars, in this case same name with constants declared above.